### ISSUE TITLE
Trouble building program.

### ISSUE THREAD

**Issue by Bartosz-Slowik** [radkum/hello_audience#1]
I attended the SFI lecture and got interested.

When i try to build i encounter this problem: 
![image](https://user-images.githubusercontent.com/92270288/233422919-be97c472-a540-4202-9a84-a1260ba40415.png)
Any suggestion  how would i resolve this?

**Comment by radkum** [comment]
Hi Bartosz,
Yes, I saw this problem. Ntapi crate has problem with latest toolchain. I'm going push a rust-toolchain.toml to set up fixed toolchain. It should solve the problem. Let me know if problem still occurs

**Comment by Bartosz-Slowik** [comment]
For some reason i still had to manually change the toolchain.
`rustup override set nightly-2022-09-23-x86_64-pc-windows-msvc `
I figured out that some tasks only work when opened via Developer command prompt VS
I also had to add registry key Debug Print filter to view messeges

Anyway it finnaly worked. Thanks a lot for this example!
![image](https://user-images.githubusercontent.com/92270288/234006733-84edfd40-698d-4e22-a875-879853e6f401.png)
